usaspending_api/etl/management/commands/update_file_c_linkages.py

```diff
@@ -26,3 +31,28 @@ def handle(self, *args, **options):
     def run_sql(self, submission=None):
         for link_type in self.LINKAGE_TYPES:
             update_c_to_d_linkages(type=link_type, submission_id=submission)
+
+    def faba_award_id_check(self):
```

Comment:
I'd also rename this method. "Check" implies some validation is occurring. It's not clear from the name that data is being updated. Maybe something like `unlink_from_removed_awards()`

Reply:
I changed this in commit 3b7e9b731d84ce83a02902478eb03a22a5e7e2b4